generate-passphrase

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+LENGTH=32
+if [ ! -z "$1" ]; then
+  LENGTH=$1
+fi
+cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9!@#$%^&*()_+?><~;' | fold -w $LENGTH | head -n 1
```

Comment:
Ah, that's better than what I provided! Also doesn't have a trailing newline, which is good.